### PR TITLE
Support Ruby 3 keyword arguments

### PR DESCRIPTION
**Pull request by mame**
This PR allows rspec to pass in Ruby 2.7.

FYI: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

### CODE REVIEW

**Comment by dentarg** [comment]
@mame don't you need to add the `ruby2_keywords` gem to [the gemspec](https://github.com/sinatra/mustermann/blob/v1.0.3/mustermann/mustermann.gemspec) too?

**Comment by mame** [comment]
@dentarg Yes, I will do later.  Thank you dor pointing that out.

**Comment by jeremyevans** [comment]
The keyword argument separation issues in mustermann that this pull request fixes are currently blocking the separation of positional and keyword arguments in Ruby 3 development (https://github.com/ruby/ruby/pull/2794).  If this pull request could be given priority, I would appreciate it.  @mame, could you add a commit that updates the gemspec?  @dentarg, is there anything else that needs to be done before this is merged?

**Comment by mame** [comment]
Sorry, I forgot it!  Will do soon

**Comment by dentarg** [comment]
@jeremyevans Not sure I can answer that, I just happened to see this one thing, other than that, I'm not really familiar with mustermann. It's @namusyaka that's been merging the most recent PRs. Perhaps @jkowens (or @zzak) can also help here?

**Comment by jkowens** [comment]
This looks good to me. I'd be fine merging it in, but I don't think I have access to do any releases. So I guess that's not super helpful 😦

Hopefully @namusyaka will have a chance to chime in.

**Comment by namusyaka** [comment]
Will do soon. Sorry for the late action.

File: mustermann/spec/mapper_spec.rb [hunk 1]
```diff
@@ -19,21 +19,14 @@
...
-      subject(:mapper) { Mustermann::Mapper.new(additional_values: :raise, "/foo" => "/bar") }
+      subject(:mapper) { Mustermann::Mapper.new({ "/foo" => "/bar" }, additional_values: :raise) }
       its(:to_h) { should be == { Mustermann.new("/foo") => Mustermann::Expander.new("/bar") } }
       example { mapper['/foo'].should be == '/bar' }
       example { mapper['/fox'].should be == '/fox' }
     end
 
     context 'allows specifying type' do
-      subject(:mapper) { Mustermann::Mapper.new(additional_values: :raise, type: :rails, "/foo" => "/bar") }
+      subject(:mapper) { Mustermann::Mapper.new({ "/foo" => "/bar" }, additional_values: :raise, type: :rails) }
```
Comment (namusyaka) [inline mustermann/spec/mapper_spec.rb]: Do we need to update an example [here](https://github.com/sinatra/mustermann/blob/d103fbdfafc4b2148c76097301341b6e006cc3e6/mustermann/lib/mustermann/mapper.rb#L43)?

**Comment by namusyaka** [comment]
@mame @jeremyevans Now the PR has been merged, and a new version including the changes has been released: https://rubygems.org/gems/mustermann/versions/1.1.0

**Pull request merged**